packages/customer-account-ui-extensions/src/components/CustomerAccountAction/CustomerAccountAction.ts

```diff
@@ -0,0 +1,22 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {ActionComponents} from 'component-set';
+
+export interface CustomerAccountActionProps {
+  /**
+   * A title rendered as a `Heading` at the top of the modal.
+   */
+  title: string;
+  /**
+   * Action modal's primary action, ie 'Save' or 'Accept'.
+   */
+  primaryAction?: ActionComponents;
```

Comment:
in admin they use `RemoteFragment`.

I wonder if we have to as well? https://github.com/Shopify/ui-extensions/blob/unstable/packages/ui-extensions/src/surfaces/admin/components/AdminAction/AdminAction.ts#L13

I'm genuinely not sure.

Reply:
changed, and keep everything same with admin 

The only issue is that with `remoteFragement`, on 3P developer side, it will always cause lint issue ;( 
<img width="850" alt="Screenshot 2023-08-18 at 17 02 31" src="https://github.com/Shopify/ui-extensions/assets/12724940/275fc0c8-1324-4889-9b08-92e485036552">